File: code/src/test/scala/cern/colt/matrix/tlong/impl/DenseColumnLongMatrix2DViewTest.scala
package cern.colt.matrix.tlong.impl

//remove if not needed
import scala.collection.JavaConversions._

class DenseColumnLongMatrix2DViewTest(arg0: String) extends DenseColumnLongMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new DenseColumnLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new DenseColumnLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new DenseColumnLongMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tlong/impl/DenseColumnLongMatrix2DViewTest.java
package cernx.colt.matrix.tlong.impl;

public class DenseColumnLongMatrix2DViewTest extends DenseColumnLongMatrix2DTest {

    public DenseColumnLongMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseColumnLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseColumnLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseColumnLongMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
